### PR TITLE
[P2022-484] storybook: three dots menu

### PR DESCRIPTION
**Pull request by jprymak**

### CODE REVIEW

**Review by PiotrAdamowicz — APPROVED** [review]
I'd suggest to add onClick action in argTypes to give access to it in story.

File: src/stories/ThreeDotsMenu.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
```
Comment (KarolCiesluk) [inline src/stories/ThreeDotsMenu.stories.tsx:1]: What do you use `ComponentMeta` for?

File: src/stories/ThreeDotsMenu.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import ThreeDotsMenu from "../components/ThreeDotsMenu/ThreeDotsMenu";
+
+import { mockMenuItems } from "../mockData/menuItems";
+
+export default {
+  title: "ThreeDotsMenu",
+  component: ThreeDotsMenu,
+};
+
+const Template: ComponentStory<typeof ThreeDotsMenu> = (args: any) => (
```
Comment (KarolCiesluk) [inline src/stories/ThreeDotsMenu.stories.tsx:11]: I am not sure if you need to set type to 'any' here…

File: src/stories/ThreeDotsMenu.stories.tsx (hunk 1 above)
Comment (jprymak) [reply src/stories/ThreeDotsMenu.stories.tsx:1]: I just copied whole line from somwhere else, it is redundant in my case

**Pull request merged**